test/integration.test.ts

```diff
@@ -1,5 +1,10 @@
 import { contains, lt } from '../client/src';
 import { XataClient } from '../codegen/example/xata';
+import dotenv from 'dotenv';
+import { join } from 'path';
+
+// Get environment variables before reading them
+dotenv.config({ path: join(process.cwd(), '.envrc') });
```

Comment:
Does this work? Because an `.envrc` file doesn't have the same format than a `.env` file.

Reply:
It does work, else I wouldn't have committed lol 🙌

Reply:
I was just double checking. We can all commit things that don't work by mistake.